spyder/widgets/mixins.py

```diff
@@ -197,12 +199,15 @@ def _format_text(self, title=None, signature=None, text=None,
         if not text:
             text = '\n<i>No documentation available</i>\n'

```

Comment:
I think it makes no sense to add the last `\n` here because it adds a blank line for no reason, right @goanpeca?

So I'd say we should remove it.